herbert_core/classes/instrument_classes/instrument/detectors/@IX_detector_bank/IX_detector_bank.m

```diff
@@ -175,7 +186,7 @@
             if all(val(:) >= 0) && all(val(:)<180)
                 obj.phi_ = val(:);
             else
-                error('Scattering angle(s) must lie in the range 0 to 180 degrees inclusive')
+                error('Scattering angle(s) must lie in the range 0 inclusive to 180 degrees exclusive')
```

Comment:
Add error ID?

Reply:
will pick up with this after the Herbert merge